components/WorkSectionAsGallery.js

```diff
@@ -45,12 +45,12 @@ const WorkGalleryAssetBlock = ({ assetBlock }) => {
       {!displayFirstAssetAsFullWidth && (
         <>
           <div className="flex col-8 pb0 md:pb1 items-end md:flex-row flex-col">
-            <div className="block md:col-4 col-8">
+            <div className="block md:col-4 col-8 md:mb0 mb_9">
               <div className="relative">
                 <Work work={works[0]} width="50vw" />
               </div>
             </div>
-            <div className="block md:col-4 col-8 md:ml_5 md:ml1">
+            <div className="block md:col-4 col-8 md:ml_5 md:ml1 md:mb0 mb_9">
```

Comment:
can we fix the conflicting classes `md:ml_5 md:ml1` here?

Reply:
yes! removed the first one!!